### PR TITLE
Update for Zewo 0.13

### PR DESCRIPTION
**Pull request by joernroeder**
Tries to make PostgreSQL conformant to Zewo 0.13.

### CODE REVIEW

File: Sources/PostgreSQL/Connection.swift [hunk 1]
```diff
@@ -184,7 +196,7 @@ public final class Connection: ConnectionProtocol {
             }
         }
 
-
+		
         guard let result:OpaquePointer = PQexecParams(
```
Comment (johnnycrab) [inline Sources/PostgreSQL/Connection.swift]: this instantiation makes problems, i.e. line 193:
`parameterData.map { UnsafePointer<Int8>($0) }`. This initialization is no longer available.

File: Sources/PostgreSQL/Result.swift [hunk 2]
```diff
@@ -85,6 +85,7 @@ public class Result: SQL.ResultProtocol {
         let start = PQgetvalue(resultPointer, Int32(rowIndex), Int32(fieldIndex))
         let count = PQgetlength(resultPointer, Int32(rowIndex), Int32(fieldIndex))
 
+		
         let buffer = UnsafeBufferPointer<UInt8>(start: UnsafePointer<UInt8>(start), count: Int(count))
```
Comment (johnnycrab) [inline Sources/PostgreSQL/Result.swift]: Same problem here with `UnsafePointer<UInt8>(start)`. Compiling error refers to the `withMemoryRebound` function

File: Sources/PostgreSQL/Connection.swift (hunk 1 above)
Comment (joernroeder) [inline Sources/PostgreSQL/Connection.swift]: this instantiation makes problems, i.e. line 193:
parameterData.map { UnsafePointer<Int8>($0) }. This initialization is no longer available.

File: Sources/PostgreSQL/Result.swift [hunk 3]
```diff
@@ -85,6 +85,7 @@ public class Result: SQL.ResultProtocol {
         let start = PQgetvalue(resultPointer, Int32(rowIndex), Int32(fieldIndex))
         let count = PQgetlength(resultPointer, Int32(rowIndex), Int32(fieldIndex))
 
+
```
Comment (joernroeder) [inline Sources/PostgreSQL/Result.swift]: Same problem here with UnsafePointer<UInt8>(start). Compiling error refers to the `withMemoryRebound` function

File: Sources/PostgreSQL/Result.swift (hunk 2 above)
Comment (joernroeder) [inline Sources/PostgreSQL/Result.swift]: Same problem here with UnsafePointer<UInt8>(start). Compiling error refers to the `withMemoryRebound` function

**Comment by Danappelxx** [comment]
I've tested on OS X and Linux - good to go! We can fix travis later (it needs postgresql to be installed and configured for the tests).

**Pull request merged**